nltk/classify/rte_classify.py

```diff
@@ -32,7 +32,7 @@ def ne(token):
         return True
     return False
 
-def lemmatize(word):
+def _lemmatize(word):
```

Comment:
@stevenbird ah yes I did mean to comment on that. As I understand it the `__init__` method of the `RTEFeatureExtractor` class is broken due to a conflict between its `lemmatize` argument and the `lemmatize` function defined here. To fix this we need to change the name of one or the other.

To get an idea of the impact I ran 2 quick QL queries to check how many function called to `lemmatize` there are in the code base as well as how many calls to them:
- https://lgtm.com/query/2024710808/project:1870011/lang:python/
- https://lgtm.com/query/2021851733/project:1870011/lang:python/

Note that the first one wast just out of curiosity/sanity check, I could have modified the second one to only show calls to the lemmatize function defined in `rte_classify`.

According to these queries the lemmatize() function is not used anywhere outside `RTEFeatureExtractor.__init__` (and the one defined in `wordnet.py` is never used) so the modification here shouldn't have undesired impact.